app/dashboard/views.py

```diff
@@ -639,16 +639,6 @@ def dashboard(request):
     }
     return TemplateResponse(request, 'dashboard/index.html', params)
 
-def ethhack(request):
-    """Handle displaying ethhack landing page."""
-
-    title = str(_(" Eth Hackathon 2019"))
-    params = {
-        'title': title,
-        'meta_description': 'Ethereal Virtual Hackathon, power by Gitcoin and Microsoft',
-    }
-    return TemplateResponse(request, 'dashboard/hackathon/ethhack_2019.html', params)
-
```

Comment:
Was this removed un-intentionally? Could we add this back in

Reply:
Fixed